release/test_bom.sh

```diff
@@ -2,7 +2,7 @@
 
 source _bom.sh
 source _liferay_common.sh
-source _test_common.sh
+source ../_test_common.sh
```

Comment:
@natocesarrego, please change files that are using the `_test_common.sh`.

Reply:
I need to introduce this change since I'm writing a new test out of the release folder. :wink: